komorebi/Cargo.toml

```diff
@@ -40,6 +40,7 @@ tracing = { workspace = true }
 tracing-appender = { workspace = true }
 tracing-subscriber = { workspace = true }
 uds_windows = { workspace = true }
+uuid = { version = "1", features = ["v4"] }
```

Comment:
Let's make this a dev dependency instead so it only compiles when running tests: https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html